src/test/resources/artifacts/ESB/connector/config/cerner.properties

```diff
@@ -0,0 +1,42 @@
+proxyDirectoryRelativePath=/artifacts/ESB/config/proxies/cerner/
+requestDirectoryRelativePath=/artifacts/ESB/config/restRequests/cerner/
+base=https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca
```

Comment:
Currently we are automating the integration test case to minimize manual modifications. So could you pls do the needful to automate this.